DigitalLearningSolutions.Data.Tests/Services/EmailServiceTests.cs

```diff
@@ -0,0 +1,296 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Threading;
+    using DigitalLearningSolutions.Data.Factories;
+    using DigitalLearningSolutions.Data.Models.Email;
+    using DigitalLearningSolutions.Data.Services;
+    using FakeItEasy;
+    using MailKit.Net.Smtp;
+    using MailKit.Security;
+    using MimeKit;
+    using NUnit.Framework;
+
+    public class EmailServiceTests
+    {
+        private EmailService emailService;
+        private IConfigService configService;
+        private ISmtpClient smtpClient;
+        private Email testEmail;
+        private Email multipleAddresseesTestEmail;
+
+        [SetUp]
+        public void Setup()
+        {
+            BodyBuilder emailTestBody = new BodyBuilder
+            {
+                TextBody = "Test body",
+                HtmlBody = @"<body style= 'font - family: Calibri; font - size: small;'>
+                                    <p>Test Body</p>
+                                </body>"
+            };
+
+            testEmail = new Email(
+                "recipient@example.com",
+                "Test Subject Line",
+                emailTestBody,
+                "cc@example.com",
+                "bcc@example.com");
+
+            multipleAddresseesTestEmail = new Email(
```

Comment:
I'm not 100% sure about defining these emails here. I feel like multiple addresses email should be created in relevant tests (e.g. multiple recipients, multiplePeopleCcd etc). Potentially having some sort of defaultEmail to use in tests that don't care which email is used is worth it though (maybe the current testEmail renamed?)

Reply:
Just to clarify, are you suggesting we should have 
- a single default Email instance defined in the SetUp that is used when testing things that aren't specifically about the addresses
- and each test about the addresses should define their own email instance?

Reply:
Yes, that's correct, although you could potentially create something similar to CurrentCourseHelper.CreateDefaultCurrentCourse, where you create the Email with defaults and only override the values that are needed for the test when calling CreateDefaultEmail.